disconnected/installing.adoc

```diff
@@ -0,0 +1,113 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-disconnected-environments"]
+= Installing a cluster in a disconnected environment
+include::_attributes/common-attributes.adoc[]
+:context: installing-disconnected-environments
+
+toc::[]
+
+You can install an {product-title} cluster in a disconnected environment, choosing the installation method and infrastructure that best suits your requirements.
+This includes installing {product-title} on either on-premise hardware or on a cloud hosting service such as Amazon Web Services (AWS).
+
+The following sections outline all of the supported methods for installing a cluster in a disconnected environment.
+
+[NOTE]
+====
+In order to learn about other requirements for installing a cluster using a particular method, be sure to review other content in the procedure's respective section of the documentation.
+====
+// Not sure if the sentence above should be a note or just plain text.
+
+ifndef::openshift-origin[]
+[id="installing-agent_{context}"]
+== Installing a cluster with the Agent-based Installer
+
+To learn more about installing a cluster in a disconnected environment with the Agent-based installer, see the following procedure:
+
+* xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-with-agent-based-installer[Installing an {product-title} cluster with the Agent-based Installer]
+endif::openshift-origin[]
+
+[id="installing-aws_{context}"]
+== Installing a cluster on Amazon Web Services
```

Comment:
```suggestion
== Installing a cluster on {aws-short}
```
Can use this since you already spelled out AWS earlier. I would recommend going through this file and using the attributes for the various platforms, such as {azure-first} and {vmw-first}.